cartographer/mapping/internal/collated_trajectory_builder.h

```diff
@@ -69,11 +70,20 @@ class CollatedTrajectoryBuilder : public TrajectoryBuilderInterface {
   void AddSensorData(
       const std::string& sensor_id,
       const sensor::FixedFramePoseData& fixed_frame_pose_data) override {
-    AddData(sensor::MakeDispatchable(sensor_id, fixed_frame_pose_data));
+    if (collate_landmarks_) {
```

Comment:
@pifon2a Is this a bug and should this be `collate_fixed_frame_`? Otherwise, could you add a comment?

Reply:
See #1331